default.json

```diff
@@ -1,39 +1,71 @@
 {
-  "extends": ["config:base", "group:monorepos", ":ignoreModulesAndTests"],
-  "schedule": ["every 2 weeks on Monday before 7am"],
-  "timezone": "Europe/Berlin",
-  "semanticCommits": "enabled",
-  "lockFileMaintenance": { "enabled": false },
-  "automergeType": "branch",
-  "packageRules": [
-    {
-      "groupName": "renovate-meta",
-      "semanticCommitType": "chore",
-      "semanticCommitScope": "deps",
-      "matchUpdateTypes": ["lockFileMaintenance", "pin"]
-    },
-    {
-      "groupName": "dependencies-non-major",
-      "semanticCommitType": "deps",
-      "automerge": false,
-      "matchDepTypes": ["dependencies"],
-      "matchUpdateTypes": ["patch", "minor"]
-    },
-    {
-      "groupName": "devDependencies-major",
-      "semanticCommitType": "chore",
-      "semanticCommitScope": "deps",
-      "automerge": false,
-      "matchDepTypes": ["devDependencies"],
-      "matchUpdateTypes": ["major"]
-    },
-    {
-      "groupName": "devDependencies-non-major",
-      "automerge": true,
-      "semanticCommitType": "chore",
-      "semanticCommitScope": "deps",
-      "matchDepTypes": ["devDependencies"],
-      "matchUpdateTypes": ["patch", "minor"]
-    }
-  ]
+	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
+
+	"extends": [
+		"config:base",
+		"group:monorepos",
+		":ignoreModulesAndTests",
+		":noUnscheduledUpdates",
+		":pinAllExceptPeerDependencies",
+		"npm:unpublishSafe"
```

Comment:
Hier entsteht ja die größte Änderung. Ich kann leider nicht die Verantwortung nehmen die Auswirkungen von https://docs.renovatebot.com/presets-default/#pinallexceptpeerdependencies ausreichend vorzusehen.

Reply:
I locked renovate to a hand full of repos. We will see how this behaves

Reply:
The prototyping team already uses version pinning on all repos.